core/util_test.go

```diff
@@ -953,7 +955,8 @@ func (b *testBroadcast) BroadcastDKG(c context.Context, p *drand.DKGPacket) erro
 		return err
 	}
 
-	dkgPacket, _ := protoToDKGPacket(p.GetDkg())
+	sch := scheme.GetSchemeFromEnv()
+	dkgPacket, _ := protoToDKGPacket(p.GetDkg(), sch)
```

Comment:
I'd add the `require.NoError(t, err)` check here.

Reply:
I'm not sure how, since the I don't have a `testing.T` here?

Reply:
added it to `testBroadcast`